.circleci/config.yml

```diff
@@ -140,50 +142,64 @@ jobs:
     steps:
     - cf_deploy:
         space: "development"
+        environment_key: "dev"
         domain_prefix: "dev"
+
   deploy_staging:
     docker:
       - image: cimg/ruby:2.7-node
     steps:
       - cf_deploy:
           space: "staging"
+          environment_key: "staging"
           domain_prefix: "staging"
 
+  deploy_production:
+    docker:
+      - image: cimg/ruby:2.7-node
+    steps:
+      - cf_deploy:
+          space: "production"
+          environment_key: "production"
+          domain_prefix: ""
```

Comment:
Isn't the domain prefix www?

Reply:
It is  yes!  🤦

Reply:
To be fair, this is something that I wanted to have a look at. It does not make sense to me that the origin needs to know what's its external name. 
I know why it's happening, but I think it should not be like this. The origin can and should be agnostic, IMO.

Reply:
Yeah